etc/skel/.config/hypr/hyprland.conf

```diff
@@ -68,289 +94,369 @@ decoration {
         ignore_opacity = false
     }
 
-    drop_shadow = false
-    shadow_range = 4
-    shadow_render_power = 3
-    shadow_ignore_window = true
+    # https://wiki.hyprland.org/0.45.0/Configuring/Variables/#shadow
+    shadow {
+        enabled = false
+        range = 4
+        render_power = 3
+        sharp = false
+        ignore_window = true
+        color = $cachydblue
+        color_inactive = $cachygray
+        offset = 1, 1
+        scale = 1.0
+    }
 
-    dim_inactive = false
-    col.shadow = rgba(1a1a1aee)
 }
 
-# Blur for waybar
-#blurls = waybar
-
+# https://wiki.hyprland.org/0.45.0/Configuring/Animations/ 
 animations {
     enabled = yes
-
-    # Some default animations, see https://wiki.hyprland.org/Configuring/Animations/ for more
-
     #bezier = myBezier, 0.05, 0.9, 0.1, 1.05
-
     bezier = overshot, 0.13, 0.99, 0.29, 1.1
-    animation = windows, 1, 4, overshot, slide
+    animation = windowsIn, 1, 4, overshot, slide
     animation = windowsOut, 1, 5, default, popin 80%
     animation = border, 1, 5, default
-    animation = fade, 1, 8, default
-    animation = workspaces, 1, 6, overshot, slide
-
+    # animation = fade, 1, 8, default
+    animation = workspacesIn, 1, 6, overshot, slide
+    animation = workspacesOut, 1, 6, overshot, slidefade 80%
     #animation = windows, 1, 7, myBezier
     #animation = windowsOut, 1, 7, default, popin 80%
     #animation = fade, 1, 7, default
     #animation = border, 1, 10, default
     #animation = workspaces, 1, 6, default
 }
 
-# See https://wiki.hyprland.org/Configuring/Dwindle-Layout/ for more
-dwindle {
-    no_gaps_when_only = false
-    force_split = 0
-    special_scale_factor = 0.8
-    split_width_multiplier = 1.0
-    use_active_for_splits = true
-    pseudotile = yes # master switch for pseudotiling. Enabling is bound to mainMod + P in the keybinds section below
-    preserve_split = yes
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#input
+input {
+    kb_layout = us,ru,br
```

Comment:
Remove ru and br

Reply:
Ok. the variant too?

Reply:
variant is fine since its commented. You can keep it.